sdk/core/perf-tests/core-rest-pipeline/test/utils/serverUrl.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const port = 8082;
```

Comment:
In general, instead of hardcoding a specific port that may already be in use and fail to bind, it's better to try random or incrementing port numbers until you find an unused one.  But if other places in JS already use hardcoded ports, it's fine to use one here as well.

Reply:
Done